modules/oe_theme_helper/src/TwigExtension/TwigExtension.php

```diff
@@ -68,6 +68,7 @@ public function getFilters(): array {
       new \Twig_SimpleFilter('to_ecl_attributes', [$this, 'toEclAttributes']),
       new \Twig_SimpleFilter('smart_trim', [$this, 'smartTrim'], ['needs_environment' => TRUE]),
       new \Twig_SimpleFilter('is_external_url', [UrlHelper::class, 'isExternal']),
+      new \Twig_SimpleFilter('filter_empty', 'array_filter'),
```

Comment:
This could lead to unwanted results:

![image](https://user-images.githubusercontent.com/153362/94517368-12114080-0228-11eb-9116-6c0c16fa1b83.png)

Let's filter when value is empty or NULL.